frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar/components/ToggleFullList.tsx

```diff
@@ -0,0 +1,35 @@
+import { c, t } from "ttag";
+
+import { Button, Group, Text } from "metabase/ui";
+
+export const ToggleFullList = ({
+  isExpanded,
+  toggle,
+  sliceLength,
```

Comment:
!nit `sliceLength` feels a bit odd to me

Reply:
I added a bit of documentation